terraform/central-account/terraform.tfvars.example

```diff
@@ -8,6 +8,7 @@ vpc_cidr            = "10.1.1.0/24"
 public_subnet_cidrs = ["10.1.1.128/28", "10.1.1.144/28"]
 private_subnet_cidrs = ["10.1.1.0/28"]
 subnet_azs          = ["us-east-1a", "us-east-1b"]
+application_admin   = "teamemail@mycompany.com"
```

Comment:
Can we use "teamemail@example.com" ? We standardize on example.com because it's the IANA standard